src/cma.ts

```diff
@@ -281,15 +238,18 @@ async function runCumulusTask(taskFunction, cumulusMessage,
     cmaStdin.write('\n<EOC>\n');
     const createNextEventOutput = await getCmaOutput(rl, errorObj);
     cmaStdin.write('\n<EXIT>\n');
+    if (isLoadNestedEventInput(createNextEventOutput)) {
+      throw new Error(`Invalid typing recieved from
+      createNextEventOutput: ${JSON.stringify(createNextEventOutput)}`);
+    }
     return createNextEventOutput;
-  }
-  catch (error) {
-    if (error.name && error.name.includes('WorkflowError')) {
+  } catch (error) {
+    if (error.name && error.name.includes('WorkflowError') && (!isCMAMessage(cumulusMessage))) {
```

Comment:
```suggestion
    if (error?.name?.includes('WorkflowError') && (!isCMAMessage(cumulusMessage))) {
```

Reply:
Hmm, fair.   Error isn't required to have name.